packages/addons/addon-depends/librespot-depends/pyalsaaudio/package.mk

```diff
@@ -0,0 +1,38 @@
+################################################################################
+#      This file is part of LibreELEC - https://libreelec.tv
+#      Copyright (C) 2017-present Team LibreELEC
+#
+#  LibreELEC is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 2 of the License, or
+#  (at your option) any later version.
+#
+#  LibreELEC is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with LibreELEC.  If not, see <http://www.gnu.org/licenses/>.
+################################################################################
+
+PKG_NAME="pyalsaaudio"
+PKG_VERSION="0.8.4"
+PKG_LICENSE="PSF"
+PKG_SITE="http://larsimmisch.github.io/pyalsaaudio/"
+PKG_URL="https://pypi.python.org/packages/52/b6/44871791929d9d7e11325af0b7be711388dfeeab17147988f044a41a6d83/pyalsaaudio-0.8.4.tar.gz"
```

Comment:
`PKG_URL=https://pypi.io/packages/source/p/$PKG_NAME/$PKG_NAME-$PKG_VERSION.tar.gz`

Rather than use the [horrible hashses](https://bitbucket.org/pypa/pypi/issues/438/backwards-compatible-un-hashed-package), a url of the following form should now work for most pypi-based packages:
`https://pypi.io/packages/source/${PKG_NAME:0:1}/$PKG_NAME/$PKG_NAME-$PKG_VERSION.tar.gz`

We'll need to update all pypi.pthon.org urls, eg. `Pillow` etc. - I'll submit a PR that updates the urls of existing Python packages.